src/lib/Theme.ts

```diff
@@ -62,7 +62,10 @@ import {
 import { simpleGit } from 'simple-git';
 import { THEME_TYPE } from '../helper/constants';
 
-shortid.characters('0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_@');
+const nanoid = customAlphabet(
```

Comment:
Did you confirm random characters count here ? How have we decided this ?

Reply:
yes if we publish 1 theme every hour ~2 thousand years or 17M IDs needed, in order to have a 1% probability of at least one collision.
by brute-force method.